### DISCUSSION TITLE
[Question]: how to define "zones" for PTZ cameras?

### DISCUSSION THREAD

**Discussion by wwhowwho** [blakeblackshear/frigate#12550] [Ask A Question]
### What is your question:

It seems to me that "zone" definitions in the config file are static coordinates w.r.t. to the frame.  But for a PTZ camera that moves around, how do you define a zone?  Its coordinate would keep changing, might even moved out of frame.  Is Frigate smart enough to dynamically adjust the zone boundaries depending on the current pan/zoom position of the camera?

Thanks!

**Comment by hawkeye217** [comment]
No, frigate does not adjust zones for PTZ cameras. Zones always assume a static scene.
